RevenueCatUI/Views/IconView.swift

```diff
@@ -35,7 +35,7 @@ struct IconView<S: ShapeStyle>: View {
 /// An icon to be displayed by `IconView`.
 enum PaywallIcon: String, CaseIterable {
 
-    case add
+    case plus
```

Comment:
Pretty sure we need to make this have the value of "add" so it codables correctly with the value we use in the JSON response for paywalls 🤔 `plus` isn't a known icon name 

```suggestion
    case plus = "add"
```

Reply:
ooff right I thought it was just referencing a local file, didn't realize it needed to match a backend name. That's a bit of a problem, given the UIKit naming conflict. 
Maybe we can solve it by decoupling the local property name from the key from the backend in the codable side, let me see

Reply:
lol just saw that you had actually already left a suggestion for the change 🤦 I'll try to repro with a paywall before and after to make sure all things work

Reply:
Be careful when you make the change that the ["+" image](https://github.com/RevenueCat/purchases-ios/blob/main/RevenueCatUI/Resources/icons.xcassets/add.imageset/add.svg) still shows up in the live paywall if you select it as the list tick. I ran into this issue during an aborted project reorg attempts.

Reply:
yeah, the image won't show up, gonna need to do something extra here, will post here again when it's solved

Reply:
Solved now. I don't _love_ the solution, but the local assets are named to match the backend exactly, and the backend sends names that might conflict with UIKit, so the solution decouples the two sides. 

![Simulator Screenshot - iPhone 15 - 2024-08-15 at 13 18 20](https://github.com/user-attachments/assets/3c2ae666-45ea-4fc3-9674-58e62bf71bc4)

Reply:
Thanks for flagging this!

Reply:
Good solution!